app/src/debug/AndroidManifest.xml

```diff
@@ -11,6 +11,13 @@
 
     <receiver android:name=".DebugNotificationActionReceiver" />
 
+    <receiver android:name=".FakeDataGenerationReceiver"
+      tools:ignore="ExportedReceiver">
```

Comment:
If we set this to `false`, then adb broadcasts will not work without also adding a permisison, afaik.

Since this is only in the `debug` builds, I thought it was ok to suppress the warning.